beetsplug/convert.py

```diff
@@ -100,8 +100,18 @@ def should_transcode(item, fmt):
             query, _ = parse_query_string(query_string, Item)
             if query.match(item):
                 return False
-    if config['convert']['never_convert_lossy_files'] and \
-            not (item.format.lower() in LOSSLESS_FORMATS):
+
+    try:
+        convert_lossy = not config['convert']['never_convert_lossy_files']
+        self._log.debug(u'Deprecated option "never_convert_lossy_files" is set.')
+    except:
+        convert_lossy = config['convert']['convert_lossy']
+    if str(convert_lossy) != str(config['convert']['convert_lossy']):
+        self._log.error(u'Opposite values in "convert_lossy" and "never_convert_lossy_files"')
+        # User can convert to lossy format after correcting the config.
+        # This is more reversible than the alternative.
+        return False
```

Comment:
I assume this should throw an error rather than copy willy-nilly.

Reply:
I think it would be fine to just let the new option override the old one. If it's set, the user presumably knows what they're doing—and I expect setting both to be very rare.